marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/hub-utils/invoke-update-node-path.mjs

```diff
@@ -0,0 +1,25 @@
+/**
```

Comment:
I think this file shouldn't be here, because we can't support .mjs files because we have to still support ML9?

Reply:
yes. it was missed between branches. I'll add a commit to address it.